wps-common/src/main/java/au/org/aodn/aws/util/EmailService.java

```diff
@@ -38,6 +38,7 @@ public void sendEmail(String to, String bccAddress,  String from, String subject
         try {
             LOGGER.info(String.format("Sending email to %s", to));
             Destination destination = new Destination().withToAddresses(to);
+            String sourceArn = WpsConfig.SOURCE_ARN;
```

Comment:
This needs reviewing by a Java person (@akashisama ?), but similarly to Cloudformation, this must always be an optional thing, and fallback to having no sourceArn if it is unset.

Reply:
```WpsConfig.SOURCE_ARN``` will just return the string "SOURCE_ARN", to get the actual ARN value you need to use ```WpsConfig.getProperty(WpsConfig.SOURCE_ARN)```

The withSourceArn method below just sets the sourceArn variable, which by default is null, so in the case of no source ARN, this variable should be set to null. If you set cloudformation parameter by default to AWS::NoValue' then I suspect nulls would get passed on through to here...